src/go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/src
+
+require (
+	github.com/ystia/yorc v3.1.2+incompatible
```

Comment:
Why do we have this ?

Reply:
genertaed automatically, don't know why, fixed as described below in next comment